babeltrace-2.yaml

```diff
@@ -0,0 +1,119 @@
+package:
+  name: babeltrace-2
+  version: 2.1.0
+  epoch: 0
+  description: "An open-source trace manipulation toolkit."
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      # needs setuptools for distuils use
+      - py3.13-setuptools
```

Comment:
Does this have a hard dependency on v3.13? Or can we use py3-setuptools? Similarly for the packages below. When python v3.14 comes along this will remain static and not get updated

Reply:
Good check, let me try with py3-setuptools

Reply:
verified and fixed all py3.13 usage to py3